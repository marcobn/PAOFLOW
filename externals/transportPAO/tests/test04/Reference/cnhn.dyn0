   6   1   1
   4
   0.000000000000000E+00   0.000000000000000E+00   0.000000000000000E+00
   0.166666666666667E+00   0.000000000000000E+00   0.000000000000000E+00
   0.333333333333333E+00   0.000000000000000E+00   0.000000000000000E+00
  -0.500000000000000E+00   0.000000000000000E+00   0.000000000000000E+00
