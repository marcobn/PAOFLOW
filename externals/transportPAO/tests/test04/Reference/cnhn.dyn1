Dynamical matrix file
                                                                           
  2    4  8  4.6685830  5.0000000  5.0000000  0.0000000  0.0000000  0.0000000
           1  'C   '    10947.356792250725     
           2  'H   '    918.68110941480279     
    1    1      0.0012979      2.4334932      2.4286151
    2    1      0.5138075      2.6964246      2.4286383
    3    2      0.0010790      1.9900921      2.4286564
    4    2      0.5138317      3.1398245      2.4286925

     Dynamical  Matrix in cartesian axes

     q = (    0.000000000   0.000000000   0.000000000 ) 

    1    1
  1.02497585  0.00000000   -0.08149235  0.00000000   -0.00005345  0.00000000
 -0.08149235  0.00000000    1.33408332  0.00000000    0.00133126  0.00000000
 -0.00005345  0.00000000    0.00133126  0.00000000    0.26631171  0.00000000
    1    2
 -0.92983792  0.00000000    0.07951955  0.00000000   -0.00003591  0.00000000
  0.07955265  0.00000000   -0.70799186  0.00000000    0.00109141  0.00000000
 -0.00004770  0.00000000    0.00132764  0.00000000   -0.18764781  0.00000000
    1    3
 -0.09953319  0.00000000   -0.00039072  0.00000000   -0.00001522  0.00000000
 -0.00450611  0.00000000   -0.60540758  0.00000000    0.00038126  0.00000000
 -0.00000392  0.00000000   -0.00021332  0.00000000   -0.08510974  0.00000000
    1    4
  0.00775134  0.00000000    0.00117495  0.00000000    0.00000011  0.00000000
 -0.00344523  0.00000000   -0.05743783  0.00000000   -0.00004622  0.00000000
 -0.00000020  0.00000000   -0.00024703  0.00000000    0.01791966  0.00000000
    2    1
 -0.92983792  0.00000000    0.07955265  0.00000000   -0.00004770  0.00000000
  0.07951955  0.00000000   -0.70799186  0.00000000    0.00132764  0.00000000
 -0.00003591  0.00000000    0.00109141  0.00000000   -0.18764781  0.00000000
    2    2
  1.02443345  0.00000000   -0.08172821  0.00000000   -0.00004987  0.00000000
 -0.08172821  0.00000000    1.33463035  0.00000000    0.00123642  0.00000000
 -0.00004987  0.00000000    0.00123642  0.00000000    0.26626326  0.00000000
    2    3
  0.00772158  0.00000000    0.00118552  0.00000000   -0.00000937  0.00000000
 -0.00270841  0.00000000   -0.05744667  0.00000000    0.00031989  0.00000000
 -0.00000204  0.00000000   -0.00021486  0.00000000    0.01791200  0.00000000
    2    4
 -0.09950596  0.00000000   -0.00019567  0.00000000    0.00000279  0.00000000
 -0.00493357  0.00000000   -0.60541734  0.00000000   -0.00010344  0.00000000
  0.00000144  0.00000000   -0.00026694  0.00000000   -0.08511574  0.00000000
    3    1
 -0.09953319  0.00000000   -0.00450611  0.00000000   -0.00000392  0.00000000
 -0.00039072  0.00000000   -0.60540758  0.00000000   -0.00021332  0.00000000
 -0.00001522  0.00000000    0.00038126  0.00000000   -0.08510974  0.00000000
    3    2
  0.00772158  0.00000000   -0.00270841  0.00000000   -0.00000204  0.00000000
  0.00118552  0.00000000   -0.05744667  0.00000000   -0.00021486  0.00000000
 -0.00000937  0.00000000    0.00031989  0.00000000    0.01791200  0.00000000
    3    3
  0.11041934  0.00000000    0.00063053  0.00000000    0.00000009  0.00000000
  0.00063053  0.00000000    0.66211974  0.00000000   -0.00010501  0.00000000
  0.00000009  0.00000000   -0.00010501  0.00000000    0.04875464  0.00000000
    3    4
 -0.01446123  0.00000000    0.00046386  0.00000000    0.00000007  0.00000000
  0.00057775  0.00000000    0.00118114  0.00000000    0.00000885  0.00000000
 -0.00000009  0.00000000   -0.00006486  0.00000000    0.02518077  0.00000000
    4    1
  0.00775134  0.00000000   -0.00344523  0.00000000   -0.00000020  0.00000000
  0.00117495  0.00000000   -0.05743783  0.00000000   -0.00024703  0.00000000
  0.00000011  0.00000000   -0.00004622  0.00000000    0.01791966  0.00000000
    4    2
 -0.09950596  0.00000000   -0.00493357  0.00000000    0.00000144  0.00000000
 -0.00019567  0.00000000   -0.60541734  0.00000000   -0.00026694  0.00000000
  0.00000279  0.00000000   -0.00010344  0.00000000   -0.08511574  0.00000000
    4    3
 -0.01446123  0.00000000    0.00057775  0.00000000   -0.00000009  0.00000000
  0.00046386  0.00000000    0.00118114  0.00000000   -0.00006486  0.00000000
  0.00000007  0.00000000    0.00000885  0.00000000    0.02518077  0.00000000
    4    4
  0.11042521  0.00000000    0.00058500  0.00000000   -0.00000003  0.00000000
  0.00058500  0.00000000    0.66213451  0.00000000    0.00007572  0.00000000
 -0.00000003  0.00000000    0.00007572  0.00000000    0.04875811  0.00000000

     Dielectric Tensor:

         58.489998485884         -0.400260756819          0.000021413833
         -0.400252212044          1.106331064066         -0.000001916732
         -0.000051865818          0.000001966563          1.070279769592

     Effective Charges E-U: Z_{alpha}{s,beta}

     atom #    1
         -1.659890993863          1.083643728622          0.002341744014
          0.001114135280          0.114769841345         -0.000066970550
         -0.000012269484          0.000278669681         -0.111188412367
     atom #    2
         -1.664192938221          1.084081061375         -0.001696533774
          0.001062802723          0.114791608767          0.000060958551
         -0.000010822500          0.000327003054         -0.111184022936
     atom #    3
         -0.074134192786         -1.091858063217         -0.000652261717
         -0.001475710272         -0.095262202390          0.000034942947
         -0.000001525381         -0.000061826097          0.131227068001
     atom #    4
         -0.072942468867         -1.091977765002          0.000011876029
         -0.001388968378         -0.095253663726         -0.000030444835
          0.000000992166         -0.000074188398          0.131232675099

     Effective Charges U-E: Z_{s,alpha}{beta}

     atom #    1
     -0.158238304720E+01     -0.217231040033E-02     -0.183417750146E-05
      0.150343330198E+01      0.951088385104E-01     -0.171478194359E-04
     -0.552247179033E-01      0.297043875509E-02     -0.112003970227E+00
     atom #    2
     -0.158694904130E+01     -0.221044979761E-02      0.825537787474E-06
      0.150290782825E+01      0.951801774603E-01      0.245651437444E-04
     -0.504420290819E-01      0.262523730905E-02     -0.112052850743E+00
     atom #    3
      0.311326605686E-01     -0.972867456760E-02     -0.512060069768E-05
     -0.884220062398E+00     -0.102981074473E+00      0.143047253721E-05
     -0.146266977147E-01      0.772677589961E-03      0.135450365549E+00
     atom #    4
      0.617297613185E-01     -0.111567860415E-01      0.248979104318E-05
     -0.884490786028E+00     -0.102964125195E+00     -0.169208520445E-04
      0.164079788709E-02     -0.121510937543E-03      0.135456924276E+00

     Diagonalizing the dynamical matrix

     q = (    0.000000000   0.000000000   0.000000000 ) 

 **************************************************************************
     omega( 1) =      -5.875087 [THz] =    -195.971801 [cm-1]
 (  0.081170  0.000000  0.491180  0.000000 -0.027186  0.000000 ) 
 (  0.081417  0.000000  0.491039  0.000000 -0.012439  0.000000 ) 
 (  0.106408  0.000000  0.488363  0.000000 -0.052958  0.000000 ) 
 (  0.110785  0.000000  0.488239  0.000000  0.015359  0.000000 ) 
     omega( 2) =       1.187645 [THz] =      39.615585 [cm-1]
 (  0.008049  0.000000  0.004471  0.000000  0.156188  0.000000 ) 
 (  0.008053  0.000000  0.004494  0.000000 -0.157713  0.000000 ) 
 (  0.008004  0.000000  0.004564  0.000000  0.688639  0.000000 ) 
 (  0.008043  0.000000  0.004606  0.000000 -0.690049  0.000000 ) 
     omega( 3) =       2.767192 [THz] =      92.303602 [cm-1]
 (  0.503187  0.000000 -0.084299  0.000000  0.016324  0.000000 ) 
 (  0.503279  0.000000 -0.084280  0.000000  0.035619  0.000000 ) 
 (  0.479186  0.000000 -0.085787  0.000000 -0.017809  0.000000 ) 
 (  0.478268  0.000000 -0.085796  0.000000  0.066118  0.000000 ) 
     omega( 4) =       4.040384 [THz] =     134.772701 [cm-1]
 ( -0.022524  0.000000  0.024501  0.000000  0.518903  0.000000 ) 
 ( -0.022521  0.000000  0.024480  0.000000  0.516483  0.000000 ) 
 ( -0.020224  0.000000  0.025003  0.000000  0.484739  0.000000 ) 
 ( -0.020023  0.000000  0.024976  0.000000  0.474068  0.000000 ) 
     omega( 5) =      26.928268 [THz] =     898.230330 [cm-1]
 (  0.000010  0.000000 -0.000183  0.000000  0.244568  0.000000 ) 
 (  0.000010  0.000000 -0.000166  0.000000 -0.244526  0.000000 ) 
 (  0.000013  0.000000 -0.000326  0.000000 -0.663585  0.000000 ) 
 (  0.000008  0.000000 -0.000348  0.000000  0.663362  0.000000 ) 
     omega( 6) =      30.538228 [THz] =    1018.645635 [cm-1]
 ( -0.000002  0.000000  0.000036  0.000000 -0.054769  0.000000 ) 
 ( -0.000003  0.000000  0.000063  0.000000 -0.054807  0.000000 ) 
 ( -0.000011  0.000000  0.000120  0.000000  0.704925  0.000000 ) 
 (  0.000002  0.000000  0.000007  0.000000  0.705037  0.000000 ) 
     omega( 7) =      34.901489 [THz] =    1164.188349 [cm-1]
 (  0.056133  0.000000  0.002559  0.000000  0.000004  0.000000 ) 
 (  0.057625  0.000000  0.003904  0.000000 -0.000000  0.000000 ) 
 ( -0.708651  0.000000  0.004603  0.000000 -0.000007  0.000000 ) 
 ( -0.700905  0.000000  0.005708  0.000000  0.000000  0.000000 ) 
     omega( 8) =      35.199949 [THz] =    1174.143908 [cm-1]
 ( -0.125214  0.000000 -0.086073  0.000000 -0.000000  0.000000 ) 
 (  0.124248  0.000000  0.085531  0.000000  0.000008  0.000000 ) 
 ( -0.679556  0.000000 -0.085219  0.000000 -0.000024  0.000000 ) 
 (  0.691356  0.000000  0.084424  0.000000 -0.000005  0.000000 ) 
     omega( 9) =      38.345717 [THz] =    1279.075455 [cm-1]
 (  0.005076  0.000000 -0.331075  0.000000 -0.000011  0.000000 ) 
 ( -0.005249  0.000000  0.331513  0.000000  0.000025  0.000000 ) 
 (  0.525909  0.000000 -0.338375  0.000000 -0.000070  0.000000 ) 
 ( -0.523879  0.000000  0.338980  0.000000 -0.000004  0.000000 ) 
     omega(10) =      46.473472 [THz] =    1550.188152 [cm-1]
 (  0.335395  0.000000 -0.069735  0.000000 -0.000001  0.000000 ) 
 ( -0.335297  0.000000  0.069474  0.000000  0.000007  0.000000 ) 
 ( -0.613564  0.000000 -0.078863  0.000000 -0.000024  0.000000 ) 
 (  0.613633  0.000000  0.078558  0.000000 -0.000006  0.000000 ) 
     omega(11) =      91.324827 [THz] =    3046.268314 [cm-1]
 (  0.001332  0.000000 -0.060465  0.000000  0.000003  0.000000 ) 
 ( -0.001351  0.000000  0.060365  0.000000  0.000006  0.000000 ) 
 (  0.000139  0.000000  0.705135  0.000000 -0.000052  0.000000 ) 
 (  0.000082  0.000000 -0.703904  0.000000 -0.000079  0.000000 ) 
     omega(12) =      92.014590 [THz] =    3069.276343 [cm-1]
 ( -0.000055  0.000000  0.058792  0.000000  0.000054  0.000000 ) 
 ( -0.000078  0.000000  0.058901  0.000000  0.000058  0.000000 ) 
 ( -0.001908  0.000000 -0.704041  0.000000  0.000231  0.000000 ) 
 ( -0.002100  0.000000 -0.705260  0.000000 -0.000099  0.000000 ) 
 **************************************************************************
