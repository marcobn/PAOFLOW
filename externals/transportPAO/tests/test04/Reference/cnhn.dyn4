Dynamical matrix file
                                                                           
  2    4  8  4.6685830  5.0000000  5.0000000  0.0000000  0.0000000  0.0000000
           1  'C   '    10947.356792250725     
           2  'H   '    918.68110941480279     
    1    1      0.0012979      2.4334932      2.4286151
    2    1      0.5138075      2.6964246      2.4286383
    3    2      0.0010790      1.9900921      2.4286564
    4    2      0.5138317      3.1398245      2.4286925

     Dynamical  Matrix in cartesian axes

     q = (   -0.500000000   0.000000000   0.000000000 ) 

    1    1
  1.60653705  0.00000000   -0.10593282  0.00000000   -0.00042098  0.00000000
 -0.10593282  0.00000000    1.23931825  0.00000000   -0.00003628  0.00000000
 -0.00042098  0.00000000   -0.00003628  0.00000000    0.22509686  0.00000000
    1    2
  0.18544528  0.00000000   -0.46502751  0.00000000   -0.00041510  0.00000000
 -0.46502701  0.00000000    0.07191676  0.00000000   -0.00000263  0.00000000
  0.00042729  0.00000000    0.00002767  0.00000000   -0.01936475  0.00000000
    1    3
 -0.10404699  0.00000000   -0.00649354  0.00000000    0.00012598  0.00000000
 -0.00458199  0.00000000   -0.60974393  0.00000000    0.00007019  0.00000000
 -0.00003732  0.00000000   -0.00005350  0.00000000   -0.06312189  0.00000000
    1    4
  0.01205187  0.00000000    0.00530725  0.00000000    0.00023878  0.00000000
 -0.10792742  0.00000000    0.00233912  0.00000000    0.00000429  0.00000000
  0.00003607  0.00000000    0.00008543  0.00000000   -0.00435936  0.00000000
    2    1
  0.18544528  0.00000000   -0.46502701  0.00000000    0.00042729  0.00000000
 -0.46502751  0.00000000    0.07191676  0.00000000    0.00002767  0.00000000
 -0.00041510  0.00000000   -0.00000263  0.00000000   -0.01936475  0.00000000
    2    2
  1.60604796  0.00000000   -0.10615156  0.00000000    0.00025811  0.00000000
 -0.10615156  0.00000000    1.23984249  0.00000000    0.00011107  0.00000000
  0.00025811  0.00000000    0.00011107  0.00000000    0.22514941  0.00000000
    2    3
  0.01194794  0.00000000    0.00529265  0.00000000   -0.00020883  0.00000000
 -0.10793129  0.00000000    0.00240731  0.00000000   -0.00000362  0.00000000
 -0.00003360  0.00000000   -0.00005609  0.00000000   -0.00437211  0.00000000
    2    4
 -0.10399940  0.00000000   -0.00635394  0.00000000   -0.00013580  0.00000000
 -0.00435895  0.00000000   -0.60974364  0.00000000   -0.00008659  0.00000000
  0.00002391  0.00000000    0.00001370  0.00000000   -0.06314046  0.00000000
    3    1
 -0.10404699  0.00000000   -0.00458199  0.00000000   -0.00003732  0.00000000
 -0.00649354  0.00000000   -0.60974393  0.00000000   -0.00005350  0.00000000
  0.00012598  0.00000000    0.00007019  0.00000000   -0.06312189  0.00000000
    3    2
  0.01194794  0.00000000   -0.10793129  0.00000000   -0.00003360  0.00000000
  0.00529265  0.00000000    0.00240731  0.00000000   -0.00005609  0.00000000
 -0.00020883  0.00000000   -0.00000362  0.00000000   -0.00437211  0.00000000
    3    3
  0.11128298  0.00000000   -0.00345002  0.00000000    0.00001191  0.00000000
 -0.00345002  0.00000000    0.65460723  0.00000000   -0.00000197  0.00000000
  0.00001191  0.00000000   -0.00000197  0.00000000    0.04313519  0.00000000
    3    4
  0.00214853  0.00000000   -0.01178544  0.00000000    0.00001966  0.00000000
 -0.01179812  0.00000000    0.00324293  0.00000000    0.00002792  0.00000000
 -0.00001848  0.00000000   -0.00002571  0.00000000   -0.00673276  0.00000000
    4    1
  0.01205187  0.00000000   -0.10792742  0.00000000    0.00003607  0.00000000
  0.00530725  0.00000000    0.00233912  0.00000000    0.00008543  0.00000000
  0.00023878  0.00000000    0.00000429  0.00000000   -0.00435936  0.00000000
    4    2
 -0.10399940  0.00000000   -0.00435895  0.00000000    0.00002391  0.00000000
 -0.00635394  0.00000000   -0.60974364  0.00000000    0.00001370  0.00000000
 -0.00013580  0.00000000   -0.00008659  0.00000000   -0.06314046  0.00000000
    4    3
  0.00214853  0.00000000   -0.01179812  0.00000000   -0.00001848  0.00000000
 -0.01178544  0.00000000    0.00324293  0.00000000   -0.00002571  0.00000000
  0.00001966  0.00000000    0.00002792  0.00000000   -0.00673276  0.00000000
    4    4
  0.11130175  0.00000000   -0.00366647  0.00000000   -0.00001248  0.00000000
 -0.00366647  0.00000000    0.65461603  0.00000000    0.00001918  0.00000000
 -0.00001248  0.00000000    0.00001918  0.00000000    0.04313867  0.00000000

     Diagonalizing the dynamical matrix

     q = (   -0.500000000   0.000000000   0.000000000 ) 

 **************************************************************************
     omega( 1) =       7.726097 [THz] =     257.714841 [cm-1]
 ( -0.000009  0.000000  0.000045  0.000000 -0.297749  0.000000 ) 
 (  0.000006  0.000000 -0.000002  0.000000 -0.297784  0.000000 ) 
 ( -0.000014  0.000000  0.000018  0.000000 -0.641336  0.000000 ) 
 (  0.000038  0.000000  0.000037  0.000000 -0.641372  0.000000 ) 
     omega( 2) =      12.239976 [THz] =     408.281643 [cm-1]
 (  0.000158  0.000000 -0.000214  0.000000  0.377907  0.000000 ) 
 ( -0.000117  0.000000  0.000227  0.000000 -0.377824  0.000000 ) 
 (  0.000510  0.000000 -0.000181  0.000000  0.597587  0.000000 ) 
 ( -0.000402  0.000000  0.000222  0.000000 -0.597766  0.000000 ) 
     omega( 3) =      14.539068 [THz] =     484.971115 [cm-1]
 (  0.138445  0.000000 -0.305759  0.000000 -0.000048  0.000000 ) 
 ( -0.141243  0.000000  0.299724  0.000000  0.000447  0.000000 ) 
 (  0.542741  0.000000 -0.300192  0.000000 -0.001640  0.000000 ) 
 ( -0.553571  0.000000  0.293994  0.000000 -0.000907  0.000000 ) 
     omega( 4) =      15.116400 [THz] =     504.228841 [cm-1]
 (  0.143847  0.000000  0.297557  0.000000 -0.000009  0.000000 ) 
 (  0.141105  0.000000  0.303534  0.000000  0.000070  0.000000 ) 
 (  0.553143  0.000000  0.296296  0.000000  0.000088  0.000000 ) 
 (  0.541802  0.000000  0.302351  0.000000 -0.000144  0.000000 ) 
     omega( 5) =      23.929226 [THz] =     798.193052 [cm-1]
 (  0.000259  0.000000  0.000417  0.000000  0.125790  0.000000 ) 
 ( -0.000248  0.000000 -0.000449  0.000000  0.125752  0.000000 ) 
 (  0.000169  0.000000  0.000478  0.000000 -0.696182  0.000000 ) 
 ( -0.000183  0.000000 -0.000486  0.000000 -0.695480  0.000000 ) 
     omega( 6) =      26.065569 [THz] =     869.453781 [cm-1]
 ( -0.000043  0.000000  0.000052  0.000000  0.092993  0.000000 ) 
 ( -0.000035  0.000000  0.000065  0.000000 -0.093090  0.000000 ) 
 ( -0.000031  0.000000  0.000016  0.000000 -0.700679  0.000000 ) 
 ( -0.000044  0.000000  0.000001  0.000000  0.701238  0.000000 ) 
     omega( 7) =      35.701491 [THz] =    1190.873545 [cm-1]
 ( -0.157837  0.000000 -0.157435  0.000000  0.000293  0.000000 ) 
 (  0.157864  0.000000  0.157426  0.000000  0.000258  0.000000 ) 
 ( -0.648719  0.000000 -0.170876  0.000000 -0.001494  0.000000 ) 
 (  0.649066  0.000000  0.171088  0.000000 -0.001588  0.000000 ) 
     omega( 8) =      39.226338 [THz] =    1308.449796 [cm-1]
 (  0.015611  0.000000 -0.104769  0.000000  0.000004  0.000000 ) 
 (  0.015835  0.000000 -0.104847  0.000000 -0.000019  0.000000 ) 
 (  0.692266  0.000000 -0.100650  0.000000 -0.000128  0.000000 ) 
 (  0.691432  0.000000 -0.100470  0.000000  0.000143  0.000000 ) 
     omega( 9) =      41.764052 [THz] =    1393.098828 [cm-1]
 ( -0.227126  0.000000 -0.001835  0.000000  0.000119  0.000000 ) 
 (  0.227655  0.000000  0.001807  0.000000  0.000102  0.000000 ) 
 (  0.668317  0.000000 -0.018684  0.000000 -0.000572  0.000000 ) 
 ( -0.670262  0.000000  0.018235  0.000000 -0.000639  0.000000 ) 
     omega(10) =      44.993350 [THz] =    1500.816605 [cm-1]
 (  0.496055  0.000000 -0.151315  0.000000  0.000011  0.000000 ) 
 (  0.495208  0.000000 -0.151351  0.000000 -0.000059  0.000000 ) 
 ( -0.438086  0.000000 -0.201528  0.000000 -0.000357  0.000000 ) 
 ( -0.435382  0.000000 -0.202003  0.000000  0.000444  0.000000 ) 
     omega(11) =      91.041567 [THz] =    3036.819779 [cm-1]
 (  0.004287  0.000000  0.060039  0.000000  0.000011  0.000000 ) 
 ( -0.004275  0.000000 -0.059911  0.000000  0.000005  0.000000 ) 
 ( -0.000281  0.000000 -0.705336  0.000000 -0.000018  0.000000 ) 
 (  0.000031  0.000000  0.703755  0.000000  0.000001  0.000000 ) 
     omega(12) =      91.449917 [THz] =    3050.440891 [cm-1]
 (  0.005134  0.000000 -0.060088  0.000000  0.000003  0.000000 ) 
 (  0.005143  0.000000 -0.060222  0.000000 -0.000005  0.000000 ) 
 ( -0.007441  0.000000  0.703697  0.000000 -0.000037  0.000000 ) 
 ( -0.007734  0.000000  0.705269  0.000000  0.000059  0.000000 ) 
 **************************************************************************
