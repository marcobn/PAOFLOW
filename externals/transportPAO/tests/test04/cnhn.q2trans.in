 &input
   fildyn='cnhn.dyn', zasr='one-dim', flfrc='cnhn.fc',
   fileout='ext.ham', onedim=.false., idir = 1
 /
