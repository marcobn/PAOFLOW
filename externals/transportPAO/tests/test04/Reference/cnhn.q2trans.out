
     Program Q2R v.5.0          starts on  1Aug2012 at 15:58: 7 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote.php

     Serial version

     reading grid info from file cnhn.dyn0
  reading force constants from file cnhn.dyn1
 macroscopic fields = T
  58.49000  -0.40026   0.00002
  -0.40025   1.10633  -0.00000
  -0.00005   0.00000   1.07028
  na=            1
  -1.65989   1.08364   0.00234
   0.00111   0.11477  -0.00007
  -0.00001   0.00028  -0.11119
  na=            2
  -1.66419   1.08408  -0.00170
   0.00106   0.11479   0.00006
  -0.00001   0.00033  -0.11118
  na=            3
  -0.07413  -1.09186  -0.00065
  -0.00148  -0.09526   0.00003
  -0.00000  -0.00006   0.13123
  na=            4
  -0.07294  -1.09198   0.00001
  -0.00139  -0.09525  -0.00003
   0.00000  -0.00007   0.13123
zasr rotation axis in 1D system=    1
Norm of the difference between old and new effective charges:    1.73582456555877073789
  nqs=            1
 q=   0.00000000  0.00000000  0.00000000
  reading force constants from file cnhn.dyn2
  nqs=            2
 q=   0.16666667  0.00000000  0.00000000
 q=  -0.16666667 -0.00000000 -0.00000000
  reading force constants from file cnhn.dyn3
  nqs=            2
 q=   0.33333333  0.00000000  0.00000000
 q=  -0.33333333 -0.00000000 -0.00000000
  reading force constants from file cnhn.dyn4
  nqs=            1
 q=  -0.50000000  0.00000000  0.00000000

      q-space grid ok, #points =    6
asr rotation axis in 1D system=    1
Norm of the difference between old and new effective charges:    0.00000000000000024851
Norm of the difference between old and new force-constants:   0.01439053967104895158
  1  1  1  1     1.362723
  1  1  1  2    -0.137994
  1  1  1  3    -0.016088
  1  1  1  4     0.000000
  1  1  1  5    -0.016088
  1  1  1  6    -0.137994
  2  1  1  1    -0.420817
  2  1  1  2    -0.608199
  2  1  1  3     0.027660
  2  1  1  4     0.000000
  2  1  1  5    -0.000817
  2  1  1  6     0.028780
  3  1  1  1    -0.101026
  3  1  1  2     0.003296
  3  1  1  3     0.001467
  3  1  1  4     0.000000
  3  1  1  5     0.001228
  3  1  1  6     0.002419
  4  1  1  1     0.007637
  4  1  1  2     0.004555
  4  1  1  3     0.004056
  4  1  1  4     0.000000
  4  1  1  5     0.004395
  4  1  1  6    -0.000519
  1  1  2  1    -0.100671
  1  1  2  2     0.017225
  1  1  2  3    -0.004283
  1  1  2  4     0.000000
  1  1  2  5     0.005026
  1  1  2  6    -0.011220
  2  1  2  1    -0.171430
  2  1  2  2     0.274596
  2  1  2  3    -0.010472
  2  1  2  4     0.000000
  2  1  2  5     0.005060
  2  1  2  6     0.013590
  3  1  2  1    -0.002856
  3  1  2  2     0.007645
  3  1  2  3    -0.002728
  3  1  2  4     0.000000
  3  1  2  5     0.001472
  3  1  2  6    -0.005264
  4  1  2  1     0.001527
  4  1  2  2    -0.003740
  4  1  2  3     0.000165
  4  1  2  4     0.000000
  4  1  2  5    -0.002520
  4  1  2  6    -0.002396
  1  1  3  1    -0.000625
  1  1  3  2    -0.000188
  1  1  3  3    -0.000120
  1  1  3  4     0.000000
  1  1  3  5    -0.000108
  1  1  3  6    -0.000244
  2  1  3  1    -0.000709
  2  1  3  2    -0.000020
  2  1  3  3     0.000088
  2  1  3  4     0.000000
  2  1  3  5    -0.000109
  2  1  3  6    -0.000295
  3  1  3  1    -0.000033
  3  1  3  2    -0.000072
  3  1  3  3     0.000052
  3  1  3  4     0.000000
  3  1  3  5    -0.000136
  3  1  3  6    -0.000171
  4  1  3  1    -0.000066
  4  1  3  2     0.000016
  4  1  3  3     0.000202
  4  1  3  4     0.000000
  4  1  3  5    -0.000021
  4  1  3  6    -0.000139
  1  2  1  1    -0.100671
  1  2  1  2    -0.011220
  1  2  1  3     0.005026
  1  2  1  4     0.000000
  1  2  1  5    -0.004283
  1  2  1  6     0.017225
  2  2  1  1    -0.171407
  2  2  1  2     0.274613
  2  2  1  3    -0.010472
  2  2  1  4     0.000000
  2  2  1  5     0.005058
  2  2  1  6     0.013593
  3  2  1  1    -0.003847
  3  2  1  2    -0.003816
  3  2  1  3    -0.000284
  3  2  1  4     0.000000
  3  2  1  5    -0.001477
  3  2  1  6     0.002790
  4  2  1  1    -0.054479
  4  2  1  2     0.054441
  4  2  1  3     0.000196
  4  2  1  4     0.000000
  4  2  1  5     0.000513
  4  2  1  6    -0.000284
  1  2  2  1     1.286379
  1  2  2  2     0.026389
  1  2  2  3     0.002859
  1  2  2  4     0.000000
  1  2  2  5     0.002859
  1  2  2  6     0.026389
  2  2  2  1    -0.302254
  2  2  2  2    -0.373785
  2  2  2  3    -0.006049
  2  2  2  4     0.000000
  2  2  2  5    -0.000058
  2  2  2  6    -0.006492
  3  2  2  1    -0.606002
  3  2  2  2     0.002300
  3  2  2  3     0.001299
  3  2  2  4     0.000000
  3  2  2  5     0.000481
  3  2  2  6     0.003223
  4  2  2  1    -0.028140
  4  2  2  2    -0.030085
  4  2  2  3    -0.001075
  4  2  2  4     0.000000
  4  2  2  5     0.000441
  4  2  2  6    -0.001028
  1  2  3  1     0.000144
  1  2  3  2    -0.000007
  1  2  3  3    -0.000050
  1  2  3  4     0.000000
  1  2  3  5     0.000106
  1  2  3  6     0.000244
  2  2  3  1     0.000032
  2  2  3  2    -0.000165
  2  2  3  3    -0.000092
  2  2  3  4     0.000000
  2  2  3  5     0.000095
  2  2  3  6     0.000203
  3  2  3  1     0.000011
  3  2  3  2    -0.000066
  3  2  3  3    -0.000061
  3  2  3  4     0.000000
  3  2  3  5     0.000027
  3  2  3  6    -0.000027
  4  2  3  1    -0.000074
  4  2  3  2    -0.000129
  4  2  3  3    -0.000072
  4  2  3  4     0.000000
  4  2  3  5     0.000018
  4  2  3  6    -0.000003
  1  3  1  1    -0.000625
  1  3  1  2    -0.000244
  1  3  1  3    -0.000108
  1  3  1  4     0.000000
  1  3  1  5    -0.000120
  1  3  1  6    -0.000188
  2  3  1  1    -0.000156
  2  3  1  2    -0.000538
  2  3  1  3    -0.000097
  2  3  1  4     0.000000
  2  3  1  5     0.000085
  2  3  1  6    -0.000058
  3  3  1  1    -0.000081
  3  3  1  2    -0.000049
  3  3  1  3    -0.000019
  3  3  1  4     0.000000
  3  3  1  5    -0.000011
  3  3  1  6    -0.000025
  4  3  1  1    -0.000042
  4  3  1  2    -0.000071
  4  3  1  3    -0.000015
  4  3  1  4     0.000000
  4  3  1  5     0.000005
  4  3  1  6    -0.000017
  1  3  2  1     0.000144
  1  3  2  2     0.000244
  1  3  2  3     0.000106
  1  3  2  4     0.000000
  1  3  2  5    -0.000050
  1  3  2  6    -0.000007
  2  3  2  1     0.000304
  2  3  2  2     0.000191
  2  3  2  3     0.000022
  2  3  2  4     0.000000
  2  3  2  5     0.000078
  2  3  2  6     0.000184
  3  3  2  1    -0.000076
  3  3  2  2     0.000035
  3  3  2  3     0.000030
  3  3  2  4     0.000000
  3  3  2  5    -0.000067
  3  3  2  6    -0.000095
  4  3  2  1     0.000082
  4  3  2  2    -0.000058
  4  3  2  3    -0.000038
  4  3  2  4     0.000000
  4  3  2  5     0.000064
  4  3  2  6     0.000080
  1  3  3  1     0.244176
  1  3  3  2     0.010173
  1  3  3  3     0.000633
  1  3  3  4     0.000000
  1  3  3  5     0.000633
  1  3  3  6     0.010173
  2  3  3  1    -0.108826
  2  3  3  2    -0.103075
  2  3  3  3     0.006314
  2  3  3  4     0.000000
  2  3  3  5    -0.002697
  2  3  3  6     0.017230
  3  3  3  1    -0.073521
  3  3  3  2    -0.003656
  3  3  3  3    -0.001561
  3  3  3  4     0.000000
  3  3  3  5    -0.001284
  3  3  3  6    -0.009588
  4  3  3  1     0.006966
  4  3  3  2     0.010544
  4  3  3  3    -0.000839
  4  3  3  4     0.000000
  4  3  3  5    -0.000538
  4  3  3  6    -0.000596
